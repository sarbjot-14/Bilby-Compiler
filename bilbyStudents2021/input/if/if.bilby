main{
	if(true){print "if worked true" $n;}
	if(false){print "if failed false" $n;}
	if(2==2){print "if worked true with equality condition" $n;}
	
	if(2==2){
	print "just checking again" $n;
	}
	
	
	

}