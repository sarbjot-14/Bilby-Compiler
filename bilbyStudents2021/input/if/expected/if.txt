if worked true
if worked true with equality condition
